### PR TITLE
Feature/collins/forgotten password

### PR DESCRIPTION
**Pull request by ONUOHA92**
adding forgotten password branch

### CODE REVIEW

File: src/components/mainComponents/forgotPassword/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                </TouchableOpacity>
+            </View>
+
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    forgotContainer: {
+        flex: 1,
```
Comment (EngrEric) [inline src/components/mainComponents/forgotPassword/ForgotPassword.js:89]: why did you do this

**Comment by EngrEric** [comment]
image

**Comment by EngrEric** [comment]
@ONUOHA92  Can you do justice to this PR

File: App.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import React from 'react'
```
Comment (ONUOHA92) [inline App.js:1]: please guy i have resolve the conflex

**Review by Okoli-Ryan — APPROVED** [review]

**Pull request merged**